AlphaWallet/Core/Coordinators/WalletBalance/WalletBalanceFetcher.swift

```diff
@@ -122,6 +122,13 @@ class WalletBalanceFetcher: NSObject, WalletBalanceFetcherType {
         for each in deletedServers {
             services.remove(at: each)
         }
+
+        queue.async { [weak self] in 
```

Comment:
Is the `queue.async` necessary?

Reply:
added it as the rest of calls of this method called async.